rasa_scheduler/requirements.txt

```diff
@@ -1,4 +1,2 @@
-celery==5.1.2
-Django==3.2.6
-django-celery-beat==2.2.1
-redis==3.5.3
\ No newline at end of file
+celery
+redis
```

Comment:
Do you not prefer to pin all the versions to whatever we know is working? Or do you intend to do that later when we have a more finished product?

Reply:
At this point in time: I prefer to pin it when we know that only that version is working. And otherwise leave it open so we always use the latest version. I prefer this instead of in the future going back and try to update everything to newer versions. I think this is a really hot debate though.